style.css

```diff
@@ -318,3 +319,105 @@ hr {
   border-bottom: 2px solid #e9e9e9;
   margin-bottom: 1.2rem;
 }
+
+/* CONTACT FORM */
+.contact-container {
+  width: 100%;
+  margin: 0 auto;
+  background: #6071ff;
+  border-top-left-radius: 5rem;
+  margin-top: -4rem;
+}
```

Comment:
- [ ] Kindly make sure to add the background image on the whole section in order to match the provided Figma design as much as possible.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/228210349-efb91860-feed-44ed-a485-6328b5c90892.png)

   - ### FIGMA DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/228210680-e5603ec6-0a74-415b-955d-07646fc587c9.png)

Reply:
I think the background image is not downloadable. I tried for several times but couldn't download it. Please If possible, you can check the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1) and confirm.

Reply:
Hello @kessie2862. Yes, the image can be downloaded. Text me on slack so I can share you the image please.

Reply:
Thanks for the feedback. I've added the background image to the contact form and I also tweaked the `font-size` of the `h2` of the form to prevent if from mixing with the white semi-circular part of the background image.

Reply:
Fixed.